lib/api/response/survey_question_response.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:survey_flutter_ic/api/response/converter/response_converter.dart';
+import 'package:survey_flutter_ic/api/response/survey_answer_response.dart';
+
+part 'survey_question_response.g.dart';
+
+@JsonSerializable()
+class SurveyQuestionResponse {
+  final String id;
+  final String? text;
+  final int? displayOrder;
+  final String? displayType;
+  final String? imageUrl;
+  final String? coverImageUrl;
+  final double? coverImageOpacity;
+  final List<SurveyAnswerResponse> answers;
```

Comment:
Should we define the list of answers nullable as well?

Reply:
Yup, @hoangnguyen92dn To be safe I think It's good to define nullable here 👍